.github/workflows/build.sh

```diff
@@ -247,7 +247,8 @@ case "$(uname)" in
   rm -r test/execute/constant_function_argument
   
   # The C code generated by Dredd may require recent C/C++ support.
-  export DREDD_EXTRA_C_ARGS="/std:c17"
+  # Dredd's prelude includes experimental `stdatomic.h`` header.
+  export DREDD_EXTRA_C_ARGS=/std:c17 /experimental:c11atomics
```

Comment:
Not quite sure what needs to be done to get this to work.

Reply:
Fixed